src/components/Editor/Properties/PropertySelectMultiple.vue

```diff
@@ -100,24 +100,51 @@ export default {
 			default: false,
 		},
 	},
+	data() {
+		return {
+			selectionData: [],
+		}
+	},
 	computed: {
+		selection: {
+			get() {
+				return this.selectionData
+			},
+			set(newValue) {
+				this.selectionData = newValue
+			},
+		},
```

Comment:
```suggestion
```

This computed property is obsolete. Instead, `this.selectionData` should be used.

Reply:
This caused linter complaints (see review from @tcitworld above).

Reply:
Ok, linter behavior seems to have changed. Removing the computed property seems fine.